main.go

```diff
@@ -21,17 +27,18 @@ func receiveAjax(w http.ResponseWriter, r *http.Request) {
 	fmt.Printf("Message body: %s\n", r.FormValue("message"))
 
 	// Send back data to client
-	fmt.Fprintf(w, "%s: <span> %s</span><br/>", r.FormValue("username"), r.FormValue("message"))
+	_, _ = fmt.Fprintf(w, "%s: <span> %s</span><br/>", r.FormValue("username"), r.FormValue("message"))
```

Comment:
It looks strange, but it is most correct as it is explicitly ignoring the returned values.